test/use-floating.test.svelte.ts

```diff
@@ -0,0 +1,29 @@
+import { describe, it, expect } from 'vitest';
```

Comment:
Just FYI, I'm personally a fan of co-locating unit tests. So we would get:

```
/hooks
  - use-floating.ts
  - use-floating.test.svelte.ts
```

I feel it encourages folks to keep tests up to date if they're at arm's reach rather than so far removed. Plus Vitest doesn't care where they're located. As long as the import paths are correct.

Reply:
I added a `test` folder because all the other ports and floating ui themselves also have a src and test folder in the root. Knowing that, you still want me to colocate them?

Reply:
In that case, follow their lead.